app/controllers/shelf_sel_searches_controller.rb

```diff
@@ -3,7 +3,7 @@
 ###
 class ShelfSelSearchesController < ApplicationController
   def delete_saved_search
-    ShelfSelSearch.find([params[:user], params[:search_name]]).delete
-    render nothing: true
+    ShelfSelSearch.find([params[:user_name], params[:search_name]]).delete
```

Comment:
Why did this param change, and does deleting a saved search form still work after this change?

Reply:
Updated for consistency, and to fix broken delete functionality:  https://github.com/sul-dlss/libsys-webforms/blob/ce60e24076a8309629cc62888cf7c310029eb48d/app/assets/javascripts/shelf_selection_reports.js#L56